configuration/package_manifests/nfpm.tedge-configuration-plugin.yaml

```
@@ -0,0 +1,69 @@
+# yaml-language-server: $schema=https://nfpm.goreleaser.com/static/schema.json
+---
+name: tedge-configuration-plugin
+description: |
+  thin-edge.io configuration management
+arch: "${PKG_ARCH}"
+platform: "linux"
+version: "${GIT_SEMVER}"
+release: "${RELEASE}"
+section: misc
+priority: "optional"
+maintainer: "thin-edge.io team <info@thin-edge.io>"
+vendor: "thin-edge.io"
+homepage: "https://thin-edge.io"
+license: "Apache-2.0"
+
+depends:
+  - tedge
+replaces:
+  - c8y_configuration_plugin
+  - c8y-configuration-plugin
```

```suggestion
  - c8y_configuration_plugin
  - c8y-configuration-plugin
  - tedge-configuration-plugin
```
Just future-proofing. The same probably applies to the `conflicts` list as well, but I'm not sure.

---

No, a package can't conflict with itself. This package definition is for the `tedge-configuration-plugin` package.

---

Aah okay. I thought this also defines which package(s) to replace during an upgrade of the `tedge-config-plugin` from one version to another.

---

The main reason for conflicts/breaks is to handle package renaming (not standard package updates to different versions). You can read more info about it here: https://wiki.debian.org/RenamingPackages